reference/forms/types/entity.rst

```diff
@@ -202,9 +202,9 @@ query_builder
 Allows you to create a custom query for your choices. See
 :ref:`ref-form-entity-query-builder` for an example.
 
-The value of this option can either be a ``QueryBuilder`` object or a Closure.
+The value of this option can either be a ``QueryBuilder`` object, a Closure or null.
 When using a Closure, you will be passed the ``EntityRepository`` of the entity
-as the only argument and should return a ``QueryBuilder``.
+as the only argument and should return a ``QueryBuilder``.  If you'd like to display a list of empty entries, you can return null in the query_builder closure.
```

Comment:
as we're in the `query_builder` documentation here, the word `query_builder` can be removed from "the query_builder closure".

Can you please also put `null` in double backticks (so it's displayed as an inline code block) and add a line break after the first word that crosses the 72th character?